classes/Product.php

```diff
@@ -6061,6 +6061,8 @@ public static function getAllCustomizedDatas($id_cart, $id_lang = null, $only_in
     }
 
     /**
+     * @deprecated 8.0.0
```

Comment:
More detailed comments would be nice i guess :thinking: (can't be sure cause i don't know code base enough ^^')

https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations

Reply:
Improved in https://github.com/PrestaShop/PrestaShop/pull/12422/commits/f1ee0055bb886d688dd124f1f594ad69973697cd